### PR TITLE
Feat/initiate corporate action and distribute

### PR DESCRIPTION
**Pull request by monitz87**
### Description

- use `initiate_corporate_action_and_distribute` extrinsic when configuring a dividend distribution
- remove the `initiateCorporateAction` procedure
- refactor some validation into the corresponding conversion utils
- minor documentation changes

### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

**Review by VictorVicente — APPROVED** [review]

File: src/api/procedures/configureDividendDistribution.ts [hunk 1]
```diff
@@ -154,6 +182,26 @@ export async function prepareConfigureDividendDistribution(
     });
   }
 
+  if (declarationDate > new Date()) {
+    throw new PolymeshError({
+      code: ErrorCode.ValidationError,
+      message: 'Declaration date must be in the past',
```
Comment (prashantasdeveloper) [inline src/api/procedures/configureDividendDistribution.ts:188]: If the message states "past", shouldn't the check be `>=` instead of `>` ?

**Pull request merged**